src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -30,9 +32,15 @@ public function get($name)
         if (0 === strpos($name, 'env(') && ')' === substr($name, -1) && 'env()' !== $name) {
             $env = substr($name, 4, -1);
 
+            // find first existing placeholder
             if (isset($this->envPlaceholders[$env])) {
                 foreach ($this->envPlaceholders[$env] as $placeholder) {
-                    return $placeholder; // return first result
```

Comment:
no need to change this one, let's reduce potential merge conflicts (so remove the previous change also)